src/components/grid-view/GridView.scss

```diff
@@ -18,16 +17,18 @@
     }
 
     &.bdl-GridView--columns-3 .bdl-GridViewSlot {
-        $columns3prct: calc(33.3333% - 6.6666px); /* 100%/3 - 20px/3 */
+        $columns3prct: calc(33.3333% - 10px); /* 100%/3 - 20px/3 */
 
         flex-basis: $columns3prct;
```

Comment:
Can we use the `flex` rule shorthand so that we can be explicit about what the flex-grow, flex-shrink values are as well? We've had flexbox issues related to that bite us in the past